lib/plugins/esbuild/index.js

```diff
@@ -24,6 +24,7 @@ class Esbuild {
     this._functions = undefined
 
     this._buildProperties = _.memoize(this._buildProperties.bind(this))
+    this._readPackageJson = _.memoize(this._readPackageJson.bind(this))
```

Comment:
I noticed that we read `package.json` multiple times, so i memoized it and replaced the functions below